### PR TITLE
Add hd path did key example

### PR DESCRIPTION
**Pull request by OR13**
Example: https://api.did.actor/

We use the same approach with https://didme.me as well.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -516,6 +516,31 @@ <h3>HD Wallets</h3>
           See <a href="https://github.com/ethereum/EIPs/issues/84">EIP84</a>
           for examples of how this is done in the ethereum community.
         </p>
+
+        <p>
+          Because HD Wallet arise from blockchain applications,
```
Comment (TallTed) [inline docs/index.html:521]:
```suggestion
          Because HD Wallet arose from blockchain applications, 
```

File: docs/index.html [hunk 2]
```diff
@@ -516,6 +516,31 @@ <h3>HD Wallets</h3>
...
+        class="example highlight"
+        title="Generating a did:key from a mnemonic and hd path"
+      >
+      mnemonic "sell antenna drama rule twenty cement mad deliver you push derive hybrid"
+      hd path "m/44'/0'/0'/0/0" 
+      key type "Ed25519" 
+      yields 'did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn'.
+      </pre>
+      <p>
+        It is not recommended to use the same root key for multiple key types, so a different mnemonic and or hd path
```
Comment (TallTed) [inline docs/index.html:541]:
```suggestion
        It is not recommended to use the same root key for multiple key types, so a different mnemonic and/or hd path 
```

**Review by peacekeeper — APPROVED** [review]
Useful explanation.

File: docs/index.html [hunk 3]
```diff
@@ -516,6 +516,31 @@ <h3>HD Wallets</h3>
...
+
+        <p>
+          We recommend using the Bitcoin HD path by default when generating key 
+          material for a DID Method that does not provide a specific recommendation for working with HD Paths.
+        </p>
+
+
+        <pre
+        class="example highlight"
+        title="Generating a did:key from a mnemonic and hd path"
```
Comment (TallTed) [inline docs/index.html:533]:
```suggestion
        title="Generating a did:key from a mnemonic and HD path"
```

File: docs/index.html [hunk 4]
```diff
@@ -516,6 +516,31 @@ <h3>HD Wallets</h3>
...
+      >
+      mnemonic "sell antenna drama rule twenty cement mad deliver you push derive hybrid"
+      hd path "m/44'/0'/0'/0/0" 
+      key type "Ed25519" 
+      yields 'did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn'.
+      </pre>
+      <p>
+        It is not recommended to use the same root key for multiple key types, so a different mnemonic and/or hd path
```
Comment (TallTed) [inline docs/index.html:541]:
```suggestion
        It is not recommended to use the same root key for multiple key types, so a different mnemonic and/or HD path 
```

**Comment by OR13** [comment]
thank you @TallTed !

**Review by mprorock — APPROVED** [review]

**Pull request merged**